Fibonacci.java

```diff
@@ -0,0 +1,16 @@
+import java.util.Scanner;
+
+public class Fibonacci {
+  public static void main(String[] args) { 
+    Scanner input = new Scanner(System.in);
+    System.out.println("Enter an integer : "); 
+    int number = input.nextInt(); 
+    double firstNumberSqrt = Math.sqrt(number * number * 5 - 4);
```

Comment:
You must put some comment that describes your implementation as it's not trivial. Anyway you do thing in invalid way - I suppose it's Binet's equation that is just approximation - you can get reliable result with it (very close but not precise). Yes, it's more effective than iteration based solution but you don't know (You never asked me or group is it ok to use rough solution, you didn't ask about it precision etc), and in that case I don't accept your solution as it's not reliable